Js/main.js

```diff
@@ -1,51 +1,25 @@
-
-//------------------------ Adding Menu list-------------------------
-
-// const ul =document.querySelector('.menu-nav ul');
-// const li=document.createElement('a');
-// const Profile =document.createElement('a');
-
-// ul.append(li);
-// ul.append(Profile);
-// Profile.innerText=('Profile'); 
-// Profile.classList.add('menu-content-list');
-// li.innerText=('contact'); 
-// li.classList.add('menu-content-list');
-
-
-
-//---------------------------- mange menu----------------------------------
-let clickhum = document.getElementById('logoburger');
-let clickclose = document.getElementById('menubackpop');
-let firstcardBack = document.getElementById('firstcardBackid');
-let mainheader=document.getElementById('mylogo-section');
-
-clickhum.addEventListener("click", function() {
-   if (firstcardBack.className = "newmenuback"){
-      mainheader.className="hide"
-      firstcardBack.className = "newmenuback";  
-      clickclose.className = "menuback";
-   }else {
-      mainheader.className="header"
-   }
-
+const clickhum = document.getElementById('logoburger');
+const clickclose = document.getElementById('menubackpop');
+const firstcardBack = document.getElementById('firstcardBackid');
+const mainheader = document.getElementById('mylogo-section');
+
+clickhum.addEventListener('click', () => {
+  if (firstcardBack.className === 'newmenuback') {
+    mainheader.className = 'hide';
+    firstcardBack.className = 'newmenuback';
+    clickclose.className = 'menuback';
+  } else {
+    mainheader.className = 'header';
+  }
```

Comment:
- [ ] Great effort working on the `mobile menu` aspect of your `portfolio` project 👏 👏, Currently, I noticed the `mobile menu` is not functioning as given in the project requirement.

- N.B:
   - When the user clicks (or taps) the hamburger button, the mobile menu appears.
   - When the user clicks (or taps) the close (X) button, the mobile menu disappears.
   - When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
   - When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.
   - The hamburger button and menu follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins).


But I noticed when the `hamburger` icon is clicked nothing appears.

Here is a video from my end 👇 


[screen-capture (1).webm](https://user-images.githubusercontent.com/78634841/210313869-acd75494-7f8a-472d-a6af-b2f8f7756ded.webm)


> Kindly check this error and fix it before submitting again for a review